crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -209,6 +216,24 @@ fn default_mem_bootstrap() -> bool {
     true
 }
 
+/// Configure Kitsune2 Reporting.
+#[derive(Clone, Default, Deserialize, Serialize, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case", rename_all_fields = "snake_case")]
+pub enum ReportConfig {
+    /// Default to no reporting.
+    #[default]
+    None,
+
+    /// Enable JsonL reporting.
+    JsonL {
```

Comment:
Perhaps this should be `JsonLines` because it will show up in YAML as `json_l` otherwise and the "l" isn't so clear as when it's run together without snake case?